### PR TITLE
fix: fix accessibility violation from CvFileUploader

### PR DESCRIPTION
**Pull request by caroluchoa**
Contributes to #1355

## What did you do?
I've removed the **tabindex** from the label tag to the div from the cv-wrapper and added a button role to that div to remove role violations.

## Why did you do it?
I did that to remove the violation reported on issue #1355. I can not just add a widget role to the label tag because it is label. So in that way, I've moved all the focus/role/tabindex to the child div.

## How have you tested it?
I've ran IBM accessibility checker and the violation was gone, with none extra violations added.

## Were docs updated if needed?

- [X] N/A
- [ ] No
- [ ] Yes

### CODE REVIEW

File: packages/core/src/components/cv-file-uploader/cv-file-uploader.vue [hunk 1]
```diff
@@ -25,6 +21,10 @@
           @dragover="onDragEvent"
           @dragleave="onDragEvent"
           @drop="onDragEvent"
+          tabindex="0"
```
Comment (caroluchoa) [inline packages/core/src/components/cv-file-uploader/cv-file-uploader.vue:24]:
```suggestion
          role="button"
          tabindex="0"
```

**Comment by lee-chase** [comment]
Unfortunately, this breaks the Carbon focus styling. It might be possible to rectify by moving the styling class down one level but this would need further verification. 

Note this component is based on https://v10-react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application

To proceed you must demonstrate the component functions in the same way and that no styling is broken.

**Comment by caroluchoa** [comment]
> Unfortunately, this breaks the Carbon focus styling. It might be possible to rectify by moving the styling class down one level but this would need further verification.
> 
> Note this component is based on https://v10-react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application
> 
> To proceed you must demonstrate the component functions in the same way and that no styling is broken.

Ok! Thanks Lee! I'm gonna try that out

**Comment by caroluchoa** [comment]
> Unfortunately, this breaks the Carbon focus styling. It might be possible to rectify by moving the styling class down one level but this would need further verification.
> 
> Note this component is based on https://v10-react.carbondesignsystem.com/?path=/story/components-fileuploader--drag-and-drop-upload-container-example-application
> 
> To proceed you must demonstrate the component functions in the same way and that no styling is broken.

Hi Lee, I have moved the label tag styling to CvWrapper component to avoid breaking focus styling you mentioned. There was the need of creating a computed property with the classname strings due to the fact that CvWrapper components does not accept Vue dynamic class binding, because the class property is created like:

```javascript
return createElement(
        context.props.tagType,
        {
          attrs: context.data.attrs,
          class: `${context.data.class || ''} ${context.data.staticClass || ''}`.trim(),
          style: { ...context.data.staticStyle, ...context.data.style },
          on: context.listeners,
        },
```
While dynamic class binding sends an object (with classes names as keys and a boolean as value) to context.data.class

**Pull request merged**